### PR TITLE
[close #153] Add serialization and deserialization operations for RawKV.KeySpanID

### LINKED ISSUE #153: cdc: data lost
### 2. Minimal reproduce step (Required)
1. start src tikv and dst tikv
2. deploy the cdc synchronization task
3. put key into src tikv
4. read key from dst tikv

### 3. What did you see instead (Required)
Can't read key from dst tikv

### 4. What did you expect to see? (Required)
read key from dst tikv

### 5. What is your migration tool and TiKV version? (Required)

- TiKV CDC:

### PR DESCRIPTION
**Pull request by zeminzhou**
### What problem does this PR solve?

When a rawkv object undergoes serialization and deserialization, its KeySpanID may become 0.

Problem Description: **TBD**

### What is changed and how does it work?

Add serialization and deserialization operations for RawKV.KeySpanID

### Code changes

- Has exported function/method change
- Has exported variable/fields change
- Has persistent data change

### Check List for Tests

This PR has been tested by at least one of the following methods:
- Unit test
- Manual test

### CODE REVIEW

File: cdc/cdc/model/kv_gen.go [hunk 1]
```diff
@@ -206,15 +212,24 @@ func (z *RawKVEntry) EncodeMsg(en *msgp.Writer) (err error) {
...
 }
 
 // MarshalMsg implements msgp.Marshaler
 func (z *RawKVEntry) MarshalMsg(b []byte) (o []byte, err error) {
 	o = msgp.Require(b, z.Msgsize())
-	// map header, size 7
+	// map header, size 8
 	// string "op_type"
-	o = append(o, 0x87, 0xa7, 0x6f, 0x70, 0x5f, 0x74, 0x79, 0x70, 0x65)
+	o = append(o, 0x88, 0xa7, 0x6f, 0x70, 0x5f, 0x74, 0x79, 0x70, 0x65)
```
Comment (haojinming) [inline cdc/cdc/model/kv_gen.go:232]: Should this be same with `EncodeMsg` func?

Comment (zeminzhou) [reply cdc/cdc/model/kv_gen.go:232]: 8 means that there are 8 fields, example https://msgpack.org/.

Comment (zeminzhou) [reply cdc/cdc/model/kv_gen.go:232]: Yes, I will modify.

**Comment by zeminzhou** [comment]
For msgp package, there is a example https://msgpack.org/

**Review by haojinming — APPROVED** [review]
LGTM

File: cdc/cdc/model/kv_gen.go [hunk 2]
```diff
@@ -234,6 +250,9 @@ func (z *RawKVEntry) MarshalMsg(b []byte) (o []byte, err error) {
 	// string "region_id"
 	o = append(o, 0xa9, 0x72, 0x65, 0x67, 0x69, 0x6f, 0x6e, 0x5f, 0x69, 0x64)
 	o = msgp.AppendUint64(o, z.RegionID)
+	// string "keyspan_id"
+	o = append(o, 0xaa, 0x6b, 0x65, 0x79, 0x73, 0x70, 0x61, 0x6e, 0x5f, 0x69, 0x64)
```
Comment (pingyu) [inline cdc/cdc/model/kv_gen.go:254]: The bytes are quite confused. Suggest to add a method to generate it.

**Review by pingyu — COMMENTED** [review]
Please add related unit test.

**Review by pingyu — COMMENTED** [review]
Rest LGTM.

File: cdc/cdc/model/kv_gen_test.go [hunk 3]
```diff
@@ -4,13 +4,46 @@ package model
...
-	v := RawKVEntry{}
+	rand.Seed(time.Now().UnixNano())
+	randomNum := rand.Uint64()
+
+	v := RawKVEntry{
+		OpType:    OpTypePut,
+		Key:       []byte("key"),
+		Value:     []byte("value"),
+		OldValue:  []byte("old_value"),
+		StartTs:   randomNum,
```
[the remark this answers is not in the document]
Comment (zeminzhou) [reply cdc/cdc/model/kv_gen_test.go]: Done

**Review by pingyu — APPROVED** [review]
LGTM~

**Pull request merged**